CodeGen/kubernetes/manifests/gaudi/codegen.yaml

```diff
@@ -243,6 +243,8 @@ spec:
           resources:
             limits:
               habana.ai/gaudi: 1
+              memory: 64Gi
+              hugepages-2Mi: 4400Mi
```

Comment:
I personally don't think adding hugepage resource requests/limits is a very good idea.  This limits the availability of the manifest file, hugepage is not a pre-request for using Gaudi. As for the issue of "RuntimeError: synStatus=26 [Generic failure] Device acquire failed.", we've met the same issue before under the following circumstance:

The k8s worker node with gaudi has both docker(and/or docker-compose) and k8s workloads requesting gaudi resources. Because docker and gaudi-k8s-device-plugin don't know each other, sometimes they will assign the same gaudi device to 2 different workload(one in docker env, one in k8s) which results such error.

Reply:
We intentionally don't have docker installed on our k8s worker to avoid the situation that you are describing where the same Gaudi device is trying to be used in two places. I checked `hl-smi` to verify that no other workload is using the devices, and retried the deployment again this morning without hugepages. It's still giving me the error. I'm not sure if there's something different with our cluster vs your cluster that is needing hugepages?

There's also another k8s deployment example using Gaudi 2 and it talks about needing hugepages: https://console.cloud.intel.com/docs/guides/k8s_guide.html#intg2-with-stable-diffusion

I reduced the hugepages limit and verified that it's still able to run. Would this be ok?